charts/dapr/crds/configuration.yaml

```diff
@@ -140,6 +140,84 @@ spec:
                 required:
                 - handlers
                 type: object
+              publishPipeline:
```

Comment:
This feature does not belong to HTTP Middleware. See https://github.com/dapr/dapr/pull/3406

Reply:
why do you say its related to http middleware ? 
also filtering can be a middleware in this pipeline, you don't a separate mechanism for that, you can have a middleware that does the filtering and condition == false short-circuits the flow ..

Reply:
An app might decide to have multiple pipelines. Can this be exported outside configuration? Maybe inside the Subscription CRD?

Reply:
well, we have two new pipelines, one the subscriber side, and on the publisher side..  (as you can see in the configuration CRD ) 

let's say the first one can be configured in the SubscriptionCRD, (as you suggested),  but the second one .. might not even have a subscription, it's just a publisher that generates new messages into some topic, so it doesn't need a subscription CRD,
we have to find another place to put it

Reply:
moved the pipeline to subscription crd